### PR TITLE
Update outline from 3.2106.3 to 3.2110.1

### PR DESCRIPTION
**Pull request by neutric**
Created with `brew bump-cask-pr`.

### CODE REVIEW

File: Casks/outline.rb [hunk 1]
```diff
@@ -16,12 +16,11 @@
 
   zap trash: [
     "~/Library/Application Scripts/6WN928JT76.com.gorillized",
-    "~/Library/Application Scripts/com.gorillized.outlinemac.standalone.extension",
-    "~/Library/Application Support/com.gorillized.outlinemac.standalone",
+    "~/Library/Application Scripts/com.gorillized.outlinemac.standalone*",
```
Comment (neutric) [inline Casks/outline.rb]: @miccal Thanks for the touch up! One accidentally got lost on the way. Reopen PR?

Comment (bevanjkay) [reply Casks/outline.rb]: @neutric You'll need to open another PR to re-add it now that this one is merged

Comment (miccal) [reply Casks/outline.rb]: Oh crap, not the first time I have done that -- `Support` and `Scripts` have the same number of letters!

Too many PR's to review.

Comment (neutric) [reply Casks/outline.rb]: All good! I can relate. Opened too many PRs and got a bit sloppy in the end. I'll fix it in a bit.

**Pull request merged**